playbook/CHANGELOG.md

```diff
@@ -3012,4 +3026,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 \* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator)*
 
 
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator)*
```

Comment:
We can clean this up in the next go around, but i try to keep these deleted... manually. @nidaqg

Reply:
Got it!